src/server/helpers/mailer.js

```diff
@@ -0,0 +1,12 @@
+import nodemailer from 'nodemailer';
+import {mailConfig} '../../../config/config.json';
+
+
+async function sendEmail({from, to, subject, html}) {
+	const transporter = nodemailer.createTransport(mailConfig);
+	// returns a promise
+	return transporter.sendMail({from, to, subject, html});
```

Comment:
nice work here, just leaving some suggestions.
as we are usng  async function we should  use the await keyword here
```suggestion
	return await transporter.sendMail({from, to, subject, html});
```
 Also the sendMail method accepts a callback function to run once the message is delivered or it failed. 
```suggestion
	return transporter.sendMail({from, to, subject, html}, (error)=>{
	  if (error) {
          return console.log(error);
      }
	}
	);
```
This callback will help us know whenever something goes wrong wilth sufficient information

Reply:
also your checks are failing because it is unable to parse `{from, to, subject, html}`. then till then you can fill it with some dummy datas to pass the checks
 {
      from: '"Nodemailer Contact" <your@email.com>',
      to: 'RECEIVEREMAILS', 
      subject: 'Node Contact Request', 
      html: 'output '
  }

Reply:
hey ! @akashgp09  thanks for your review !
first for the async part , i think it not necessary to use `await` here, async definitions are just meant to declare that this fucntion is aynchronous in nature , and returning a promise from such functions means the same.
when this sendEmail() will be invoked then we can use `await` to to see it that is resolved or not.
returning simple promise abstracts erro handlling to the invokation place and this way we get clear API .

secondly ,i dont think we need to add `error` callback function, as all those error will ultimately be cathed in promise object !

third part , i was wondering equivalent as lint error information is not adequate to trace the error, but  let me try this sugestion though!

Reply:
may be this is what i am intending ! 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#return_value
@MonkeyDo  what are your thoughts on this

Reply:
Returning a promise from an async function is fine, as you mention it's equivalent to `await`ing it. Per the nodemailer docs:
>If callback argument is not set then the method returns a Promise object.

We'll end up using `sendMail` as a promise.
We can then proceed normally and wrap it in a try-catch block if we use it with async/await syntax, or with `.catch` if we're using promise syntax.

Let's start with that, and depending on how we use it we may decide to instead use a callback to be able to log error here.